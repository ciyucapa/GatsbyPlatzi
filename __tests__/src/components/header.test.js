import React from "react"
import renderer from "react-test-renderer"
import Header from '../../../src/components/header'

describe("Header", () => {
    it("renders correctly", () => {
      const tree = renderer
        .create(<Header 
                    cart={[]}
                />)
        .toJSON()
      expect(tree).toMatchSnapshot()
    });
});