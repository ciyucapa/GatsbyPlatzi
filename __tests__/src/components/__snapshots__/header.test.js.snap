// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="sc-bdVaJa kdaubh"
>
  <a
    href="/"
  >
    <img
      alt="logo swag"
      src="https://i.postimg.cc/6q3pg48v/Logo.png"
    />
  </a>
  <nav>
    <ul>
      <li
        className="sc-EHOje cVIwKk"
      >
        <a
          href="/"
        >
          Productos
        </a>
      </li>
      <li
        className="sc-EHOje cVIwKk"
      >
        <a
          href="http://www.platzi.com"
        >
          Platzi
        </a>
      </li>
      <li
        className="sc-EHOje hQmllw"
      >
        <a
          href="/cart"
        >
          <span>
            <img
              alt="cart logo"
              src="https://i.postimg.cc/L6wpMxLt/cart.png"
            />
          </span>
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
